docker/maistra-builder_2.1.Dockerfile

```diff
@@ -47,7 +47,7 @@ RUN curl -sfL https://download.docker.com/linux/fedora/docker-ce.repo -o /etc/yu
 ENV GOBIN=/usr/local/bin
 RUN GO111MODULE=off go get github.com/myitcv/gobin && \
     gobin github.com/jstemmer/go-junit-report@${GO_JUNIT_REPORT_VERSION} && \
-    gobin github.com/mikefarah/yq/v3 && \
+    gobin github.com/mikefarah/yq/v4 && \
```

Comment:
I think @luksa said there were issues created by changes between v3 and v4 right? Have those been fixed already?